src/utils/getUser.ts

```diff
@@ -1,13 +1,20 @@
-import { getAccessToken } from "./getCookie";
-import { type AsyncActionResponse, handleAsync } from "./handleAsync";
-import { GET } from "./requests";
-import { type User } from "@/store/features/user/userSlice";
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-explicit-any */
 
-export function getUser(): Promise<AsyncActionResponse<User>> {
-  const token = getAccessToken();
+import { axiosInstance } from "./axiosInstance";
 
-  const getUserAsync = () =>
-    GET<User>("api/v1/users/me", token, "force-cache", "me");
+export async function getUser() {
```

Comment:
Are we going to create a module also for this logic following the hexagonal pattern but in another pull request?